src/args.rs

```diff
@@ -51,21 +51,22 @@ pub struct CliArguments {
     pub input: PathBuf,
 }
 
+static NONE: &str = "None";
 static LONG_VERSION: Lazy<String> = Lazy::new(|| {
     format!(
         "
 Build Timestamp:     {}
 Build Git Discribe:  {}
```

Comment:
let's also fix this typo

```suggestion
Build Git Describe:  {}
```